### DISCUSSION TITLE
How to save the torch model trained by PINNs?

### DISCUSSION THREAD

**Discussion by yfche** [rezaakb/pinns-torch#5] [Q&A]
I am following the tutorials for navier stokes equation, and I wonder how to save the trained torch model such that it can be loaded to other projects for making predictions. Any hands-on instruction would be much appreciated. Thanks.

**Comment by rezaakb** [comment]
You can enable checkpointing by passing the argument `enable_checkpointing=True` to the trainer. This can be done by including it in the config file or directly defining it when you create the trainer. For more information, please refer to this [link](https://lightning.ai/docs/pytorch/stable/common/checkpointing_basic.html). Thank you!
